.github/workflows/build.yml

```diff
@@ -36,7 +36,7 @@ jobs:
 
       # temporarily set --guardiand_loglevel=info to debug https://github.com/wormhole-foundation/wormhole/issues/3052
       - run: tilt ci -- --ci --namespace=$DEPLOY_NS --num=2 --guardiand_loglevel=info
-        timeout-minutes: 25
+        timeout-minutes: 60
```

Comment:
bumped this after a timeout but the following run finished in 18m 

leave it or revert?